engine/swarm/swarm-tutorial/index.md

```diff
@@ -71,12 +71,29 @@ The tutorial uses `manager1` : `192.168.99.100`.
 
 The following ports must be available. On some systems, these ports are open by default.
 
-* **TCP port 2377** for cluster management communications
-* **TCP** and **UDP port 7946** for communication among nodes
-* **UDP port 4789** for overlay network traffic
+* Port `2377` TCP for communication with and between manager nodes
+* Port `7946` TCP/UDP for overlay network node discovery
+* Port `4789` UDP (configurable) for overlay network traffic
 
 If you plan on creating an overlay network with encryption (`--opt encrypted`),
-you also need to ensure **ip protocol 50** (**ESP**) traffic is allowed.
+you also need to ensure **IP protocol 50** (**IPSec ESP**) traffic is allowed.
+
+Port `4789` is the default value for the Swarm Data Path port, also known as the VXLAN port.
```

Comment:
```suggestion
Port `4789` is the default value for the Swarm data path port, also known as the VXLAN port.
```

Reply:
The title casing is meant to be similar to the output of `docker info` -- where it is
```
Swarm:
    Data Path port:
```

However, that's a bit obscure/probably not necessary, in hindsight.

Reply:
ah, yeah, tbh the casing on `docker info` seems all over the place in general, a do-over would be nice.